src/Corporation/Corporation.ts

```diff
@@ -181,7 +181,7 @@ export class Corporation {
 
   determineCycleValuation(): number {
     let val,
-      assetDelta = this.totalAssets - this.previousTotalAssets;
+      assetDelta = (this.totalAssets - this.previousTotalAssets)/10;
```

Comment:
Where does this factor of 10 come from? I think it's based on `corpConstants.gameCyclesPerCorpStateCycle`, so better to use that directly.

Reply:
oh you already did 😆